Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -540,17 +376,28 @@ public function flushConfigurationCache()
         }
     }
 
+    /**
+     * Generate configuration with environment variables replaced without modifying or loading the cache
+     * @param string $configurationType
+     */
+    protected function processConfiguration(string $configurationType)
+    {
+        $config = null;
+        eval('$config = '.$this->replaceVariablesInPhpString(var_export($this->unprocessedConfiguration[$configurationType], true)).';');
```

Comment:
My security sensors tingle with this eval, but I can't put my finger on a valid abuse-case, since the var_export somewhat limits what is possible on the right side of the assignment - if I see correctly, s.th. like `''; do_something_evil();` shouldn't be possible

Reply:
I've refactored the `eval` and now use a temporary file to include. The tests are now failing again - but because of some other issue with the master I guess.